web/src/components/CategoryListItem.tsx

```diff
@@ -1,62 +1,39 @@
+import Avatar from '@mui/material/Avatar'
 import Divider from '@mui/material/Divider'
+import ListItem from '@mui/material/ListItem'
+import ListItemAvatar from '@mui/material/ListItemAvatar'
+import ListItemButton from '@mui/material/ListItemButton'
+import ListItemText from '@mui/material/ListItemText'
+import Typography from '@mui/material/Typography'
 import { styled } from '@mui/material/styles'
 import React, { ReactElement } from 'react'
-import { Link } from 'react-router-dom'
 
 import { CategoryModel } from 'shared/api'
 
-const Row = styled('li')`
-  width: 100%;
-`
+import Link from './base/Link'
+import List from './base/List'
 
-const SubCategoriesContainer = styled('ul')`
-  list-style-type: none;
+const StyledList = styled(List)`
+  padding: 0;
 `
 
-const SubCategory = styled('li')`
-  text-align: start;
-  width: 100%;
+const StyledListItem = styled(ListItem)`
+  flex-direction: column;
+  align-items: stretch;
 `
 
-const CategoryThumbnail = styled('img')`
-  width: 30px;
-  height: 30px;
-  padding: 0 5px;
-  flex-shrink: 0;
-  object-fit: contain;
-  filter: ${props => (props.theme.isContrastTheme ? 'invert(1)' : 'none')};
+const SubCategoryListItem = styled(ListItem)`
+  padding: 0;
 `
 
-const CategoryItemCaption = styled('span')`
-  align-items: center;
-  padding: 15px 5px;
-  color: inherit;
-  font-size: ${props => props.theme.legacy.fonts.contentFontSize};
-  font-weight: bold;
-  text-decoration: inherit;
-  height: 100%;
-  min-width: 1px; /* needed to enable line breaks for too long words, exact value doesn't matter */
-  flex-grow: 1;
-  word-wrap: break-word;
-`
+const StyledSubCategoryListItemButton = styled(ListItemButton)`
+  padding-left: 72px;
+  min-height: 56px;
```

Comment:
❓ Personal opinion, I think it looks better if the sub category list items are a little bit shorter without the min-height and it takes less space. Also, the distinction between categories and sub categories gets bigger through this.
```suggestion
```

Reply:
@hauf-toni what do you think:
<p float="left">
<img width="200" height="1664" alt="image" src="https://github.com/user-attachments/assets/0ce3e43c-659d-4305-ba74-6b80b61995da" />
<img width="200" height="1664" alt="image" src="https://github.com/user-attachments/assets/21216b30-0828-4a9e-83a0-ecee091a41f7" />
</p>

Reply:
hmm  I can see there is padding on the list other than the height difference...

Reply:
Yes, without a min-height this would also need to be removed again as you did before.

Reply:
on small screens without `min-height: 56px;` we will end up with 3 different sizes instead of 2
<img width="400" alt=";ist" src="https://github.com/user-attachments/assets/003753d0-8340-47ac-8f1f-0e6bbd818f74" />

Reply:
Personally I think its fine that longer items just take more space.